query-string/index.d.ts

```diff
@@ -13,7 +13,7 @@ declare module "query-string" {
      * Leading ? or # are ignored, so you can pass location.search or location.hash directly.
      * @param str
      */
-    export function parse(str: string): { [key: string]: string | string[] };
+    export function parse(str: string): { [key: string]: string | string[] | null };
```

Comment:
As far as I see `undefined` isnt a valid value.

On the other hand, current definition does not compile using `strictNullChecks` which is a must have. isnt it?